subsys/bluetooth/mesh/Kconfig

```diff
@@ -380,6 +380,17 @@ endchoice
 
 if BT_MESH_ADV_EXT
 
+config BT_MESH_ADV_MAIN_RELAY_ENABLED
+	bool "Advertiser main support relay messages"
+	default y if !BT_MESH_FRIEND && !BT_MESH_LOW_POWER
+	depends on BT_MESH_RELAY
+	help
+	  Enable advertiser main to support relay messages, this maximizes the
+	  utilization efficiency of advertising sets, which is helpful for the
+	  sending of dense relays, but for Friend and LPN, it delays the sending
+	  of local messages a little, this may cause the time slot deviation between
+	  friend and lpn, and the power consumption of lpn will increase.
```

Comment:
```suggestion
	  When this option is enabled, there is a message that needs to be
	  relayed, all relay advertising sets defined by
	  CONFIG_BT_MESH_RELAY_ADV_SETS are busy with relaying messages
	  and the main advertising set is not busy with sending local
	  messages, the stack will use the main advertising set to relay
	  the message. This maximizes the utilization efficiency of
	  advertising sets, which is helpful for the sending of dense
	  relays. With CONFIG_BT_MESH_RELAY_RETRANSMIT_COUNT value
	  greater than zero, this can noticeably delay transmission of
	  local messages. When Friend feature is enabled and the node is
	  in a friendship, this option can delay transmission of local
	  messages thus requiring bigger CONFIG_BT_MESH_FRIEND_RECV_WIN
	  value. This in turn will result in increase of the power
	  consumption of the Low Power node.
```